lib/puma/plugin/tailwindcss.rb

```diff
@@ -0,0 +1,65 @@
+require "puma/plugin"
+
+Puma::Plugin.create do
+  attr_reader :puma_pid, :tailwind_pid, :log_writer
+
+  def start(launcher)
+    return unless Rails.env.development?
```

Comment:
Wouldn't this condition be better in the Puma configuration? Only include the plugin if the environment is a development environment.

Reply:
Yea i went back and forth. I choose this one so people didnt have to worry about it in the puma config but i think having it be explicit in the puma config gives some flexibility. Changed it